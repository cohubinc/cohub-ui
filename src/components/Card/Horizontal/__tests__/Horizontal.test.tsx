import React from "react";
import { mount } from "enzyme";

import { Card } from "dist";

describe("Horizontal", () => {
  it("renders without crashing", () => {
    const wrapper = mount(<Card.Horizontal title="test" />);
    expect(wrapper).toMatchSnapshot();
  });
});
