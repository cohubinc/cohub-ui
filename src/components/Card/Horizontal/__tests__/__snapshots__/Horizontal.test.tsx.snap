// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Horizontal renders without crashing 1`] = `
<Horizontal
  elevation={1}
  title="test"
>
  <div
    className="Horizontal-module_CardHorizontal__2b_IU undefined"
    style={
      Object {
        "boxShadow": "0px -1px 1px rgba(0, 0, 0, 0.03), 0px 1px 1px rgba(0, 0, 0, 0.14)",
      }
    }
  >
    <div
      className="flex w-100"
    >
      <div
        className="flex w-100 ml-1"
      >
        <div
          className="ml-1 w-100"
        >
          <Component
            block={true}
          >
            <span
              className=""
              data-qa="heading-tiny"
              style={
                Object {
                  "color": "hsl(0, 2%, 22%)",
                  "display": "block",
                  "fontFamily": "Inter",
                  "fontSize": 18,
                  "fontWeight": 400,
                  "textAlign": "initial",
                  "textTransform": "initial",
                }
              }
            >
              test
            </span>
          </Component>
          <Typography
            block={true}
          >
            <Component
              block={true}
            />
          </Typography>
        </div>
      </div>
    </div>
  </div>
</Horizontal>
`;
